packages/scss/src/components/inputs/textfield/_textfield.framed.scss

```diff
@@ -60,74 +60,23 @@
 	.textfield-input {
 		width: 100%;
 		border-radius: 0;
-		background-color: _color("white");
 		box-shadow: fakeborderoverlay(_component("field.framed.border"));
 		padding: _component("field.framed.top-padding") _component("field.framed.side-padding") _component("field.framed.bottom-padding");
 		line-height: _theme("sizes.standard.line-height");
 
-		&:not(:disabled) {
-			&:not(.is-error) {
-				&:not(.is-success) {
-					&:not(.is-valid) {
-						&:not(.is-warning) {
-							&:not(.is-invalid) {
-								&:not(.ng-invalid) {
-									background-color: white;
-
-									&:hover {
-										background-color: _component("field.framed.color50");
-									}
-
-									&:focus {
-										background-color: white;
-										box-shadow: fakeborderoverlay(_component("field.framed.color")), 0 0 0 4px _component("field.framed.color50");
-									}
-								}
-							}
-						}
-					}
-				}
-			}
-
-			&.is-error {
-				box-shadow: fakeborderoverlay(_color("error"));
-				z-index: 3;
+		&:not(:disabled, .is-error, .is-success, .is-valid, .is-warning, .is-invalid, .ng-invalid) {
```

Comment:
&:not(:disabled):not(.is-disabled) à rajouter ?

Reply:
The `.is-disabled` class doesn't work on the framed form elements (styles are overridden by the more specific framed style declarations). I feel like this is a bad practice in any case and the `.is-disabled` class should be deprecated in favour of the `disabled` attribute.